webapp/src/pages/applicationForm/components/ApplicationForm.js

```diff
@@ -1009,14 +1010,22 @@ class ApplicationListComponent extends Component {
     }
   }
 
+  componentDidMount() {
+    eventService.getEvent(this.props.formSpec.event_id).then(response => {
```

Comment:
We do pass event as a prop to the ApplicationForm (from [EventHome.js](https://github.com/deep-learning-indaba/Baobab/blob/c4dd0240a4182ad8980a8abd2c9b38ccd7c7b2cc/webapp/src/pages/eventHome/EventHome.js#L153)). Can we not just get event_type from there instead of having to do an extra call to the backend?